src/js/resize-manager.js

```diff
@@ -59,21 +59,22 @@ class ResizeManager extends Component {
     this.resizeObserver_ = null;
     this.debouncedHandler_ = debounce(() => {
       this.resizeHandler();
-    }, 100, false, player);
+    }, 100, false, this);
 
     if (RESIZE_OBSERVER_AVAILABLE) {
       this.resizeObserver_ = new this.ResizeObserver(this.debouncedHandler_);
       this.resizeObserver_.observe(player.el());
 
     } else {
       this.loadListener_ = () => {
-        if (this.el_.contentWindow) {
-          Events.on(this.el_.contentWindow, 'resize', this.debouncedHandler_);
+        if (!this.el_ || this.el_.contentWindow) {
+          return;
         }
-        this.off('load', this.loadListener_);
+
+        Events.on(this.el_.contentWindow, 'resize', this.debouncedHandler_);
       };
 
-      this.on('load', this.loadListener_);
+      this.one('load', this.loadListener_);
```

Comment:
This may be a fix, I doubt we are seeing two load events, but if we were. This would have caused two debounce handlers to be added for window `resize`.